<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>627</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Kryptografia i teoria kodów - projekt  (Podpis cyfrowy)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="select_file">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>480</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Wybierz plik PDF</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>481</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Nie wybrano pliku</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="generate_keys">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>480</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Wygeneruj klucze</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sing_file">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>480</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Podpisz plik PDF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="verify_signature">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>480</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Zweryfikuj podpis</string>
    </property>
   </widget>
   <widget class="QPushButton" name="load_certificate">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>480</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Przeglądaj certyfikat X.509</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="certificate_info">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>481</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>481</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="load_chain">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>480</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Przeglądaj łańcuch certyfikatów</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
